pkg/ui/config/config.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) The Thanos Authors.
```

Comment:
I still think this file can be eliminated by introducing a function to bucket config. https://github.com/thanos-io/thanos/blob/97596d9a97113495a453261670804c7c67c681e7/pkg/objstore/client/factory.go